src/openforms/js/components/admin/debug/definition.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
this is an included sample file of the schema of such a complex variable definition. It serves no other purpose than documentation.